test/myFTClient.spec.js

```diff
@@ -835,172 +835,197 @@ describe('myFT Client proxy', () => {
 		});
 	});
 
-		//new orgainic v3/kat methods
-		describe('Followed_by_Kat concepts', () => {
-			const relProps = Object.assign({}, myFT.followedProperties, {byTool: 'myFTClient.spec', isTest: true});
-			const {ids, subjects} = require('./mocks/fixtures/userFollowsConceptByKat');
-			const followedByKatRes = require('./mocks/fixtures/followByKatResp');
+	//v3/kat remove users endpoint
+	describe('Remove user from a licence', () => {
 
-			afterEach(done => {
-					nock.cleanAll();
-					done();
-			});
+		afterEach(done => {
+			nock.cleanAll();
+			done();
+		});
 
-			it('Should set concept(s) follows by a user ', done => {
-					nock(baseUrl)
-						.post('/kat/user/follows')
-						.query(true)
-						.reply(200, () => followedByKatRes);
+		it('should return 204 with empty object when calling /kat/user/remove remove all of a users relationships related to a given licence', () => {
 
-				myFT.addConceptsFollowedByKatUser(ids, subjects, relProps)
-					.then(addResp => {
-						expect(addResp).to.be.an('array');
-						expect(addResp[0][0][0]).to.have.deep.property('katRel.type', 'followed_by_kat');
-						done();
-					})
-					.catch(done);
+			nock(baseUrl)
+			.delete('/kat/user/remove')
+			.query(true)
+			.reply(204);
+
+			return myFT.removeKatUsersFromLicence (uuids.validLicence, uuids.validUser)
+			.then((resp) => {
+				expect(resp).to.be.an('Object');
+				expect(resp.status).to.equal(204);
 			});
 
-			it('Should get an array of concepts followed by a group', done => {
-					nock(baseUrl)
-						.get(`/group/${uuids.validLicence}/followed_by_kat/concept?page=1&limit=500`)
-						.reply(200, () => require('./mocks/fixtures/groupFollowedConcept'));
-
-
-				myFT.getConceptsFollowedByKatGroup(uuids.validLicence)
-					.then(followResponse => {
-						expect(followResponse).to.be.an('array');
-						expect(followResponse).to.have.lengthOf(1);
-						expectOwnProperties(followResponse, ['uuid']);
-
-						done();
-					})
-					.catch(done);
-			});
+		});
 
-			it('Should set concept(s) follows by a group ', done => {
+	});
 
-					nock(baseUrl)
-						.post('/kat/group/follows')
-						.query(true)
-						.reply(200, () => followedByKatRes);
+	//new orgainic v3/kat methods
+	describe('Followed_by_Kat concepts', () => {
+		const relProps = Object.assign({}, myFT.followedProperties, {byTool: 'myFTClient.spec', isTest: true});
```

Comment:
Sending `isTest: true` seems a bit code smelly. The thing you're testing shouldn't be aware it's under test. We ain't Volkswagen